src/animation/keyframe.cpp

```
@@ -25,22 +25,22 @@ namespace nlohmann {
     void to_json(nlohmann::json& json, Keyframe const& keyframe) {
-        json["frame"] = keyframe.m_frame;
-        json["value"] = keyframe.m_value;
-        json["interp"] = keyframe.m_interpType;
+        json["frame"] = keyframe.frame;
+        json["value"] = keyframe.value;
+        json["interp"] = keyframe.interpType;
     }
 
     void from_json(nlohmann::json const& json, Keyframe& keyframe) {
-        nlohmann::json valueJson = json.at("value");
+        nlohmann::json valueJson = json.value("value", nlohmann::json{});
```

Clang (14.0.6)

[error] variable 'valueJson' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)